akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/common/TapiporlaConfig.scala

```diff
@@ -0,0 +1,31 @@
+package com.tapiporla.microservices.retrievers.common
+
+import java.util.concurrent.TimeUnit
+
+import com.typesafe.config.{Config, ConfigFactory}
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+object TapiporlaConfig {
+
+  private val config: Config = ConfigFactory.load()
+
+  private val daemonConfig = config.getConfig("tapiporla.daemon")
```

Comment:
why private?

Reply:
I want to forbid the use of intermedial configs.
If you want to get a property, you have to use the method (for example):
TapiporlaConfig.ElasticSearch.timeBeforeRetries

Reply:
I see your point. You're right